.github/workflows/release-snapshot.yml

```diff
@@ -29,6 +29,11 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+      - name: Verify NPM Token is valid
+        run: npm whoami
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
```suggestion
```
This isn't necessary because the previous step sets the npm config I only added it to be extra paranoid that the test was a valid test